packages/turf-meta/index.js

```diff
@@ -98,7 +98,7 @@ export function coordEach(geojson, callback, excludeWrapCoord) {
                         callback(coords[j][k], coordIndex, featureIndex, featureSubIndex);
                         coordIndex++;
                     }
-                    if (geomType === 'MultiLineString') featureSubIndex++;
```

Comment:
Strange line 🤔 Don't know why this was here in the first place...

There's a bunch more like that...

Reply:
Actually this is there for a reason, `featureSubIndex` is used to find the MultiPart Index.

If you wanted to know which Ring from the Polygon we would need 3 tiers.

- FeatureIndex
- FeatureSubIndex
- GeometryIndex (currently not implemented)